components/KeyFactIndicator/KeyFactIndicator.tsx

```diff
@@ -54,9 +57,10 @@ const KeyFactIndicator: FunctionComponent<KeyFactIndicatorProps> = ({ indicator,
                             stat.dataFormat === 'currency' && props.useLocalValue
                               ? props.currencyCode
                               : stat.valuePrefix || indicator.value_prefix,
-                          suffix: stat.valueSuffix || indicator.value_suffix,
+                          suffix: suffix,
                           dataFormat: stat.dataFormat || indicator.data_format,
-                          aggregation: stat.aggregation
+                          aggregation: stat.aggregation,
+                          decimalCount: setDecimalCount(suffix)
```

Comment:
stat (IndicatorStat) should also include a `decimalCount` option, which is then passed as a default to `setDecimalCount`

Reply:
Added `decimalCount` option to `IndicatorStat`